### ISSUE TITLE
No way to set autopilot airspeed through external controls

### ISSUE THREAD

**Issue by slemson** [flybywiresim/a32nx#5898]
### Aircraft Version

Stable

### Build info

```json
{
  "built": "2021-09-18T00:34:38+00:00",
  "ref": "v0.7.0",
  "sha": "e8ea73b3648d48437ca5df55e280a945fd11dab6",
  "actor": "Benjozork",
  "event_name": "manual"
}
```

### Describe the bug

As I change the speed selector in the cockpit, if I watch any of the following data:

- LVAR:A32NX_AUTOPILOT_SPEED_SELECTED
- MSFS:AUTOPILOT AIRSPEED HOLD VAR:2
- MSFS:AUTOPILOT AIRSPEED HOLD VAR:1
- SIMCONNECT:AUTOPILOT AIRSPEED HOLD VAR

I see the values of the data changing as I move the cockpit dial.  However, when using SPAD.neXt to map changing any of those data points while spinning my real cockpit dial, the data values programmatically change, but the cockpit value does not. If I monitor actions, spinning the change dial on the speed selector in cockpit does not trigger any action that SPAD.neXt can see.

In other words, I can read the set airspeed data from the cockpit into SPAD.neXt, but I can't set it that way.  Every other control allows this but the speed dial. For example, incrementing/decrementing AUTO PILOT ALTITUDE LOCK VAR does reflect in the cockpit AND my real gauge.

### Expected behavior

I would expect that incrementing or decrementing AUTOPILOT AIRSPEED HOLD VAR would change the value in the FCU in the cockpit.

### Steps to reproduce

1. Using an external controller, increase or decrease the value of the AUTOPILOT AIRSPEED HOLD VAR data point
2. Observe that the cockpit value does not change.

### Discord Username (optional)

Thirdwaver#8452

**Comment by aguther** [comment]
https://docs.flybywiresim.com/fbw-a32nx/a32nx-lvar-events/

**Comment by slemson** [comment]
I apologize.  I experimented for over an hour, read forum posts, googled other sites and couldn't find the answer.  I spent a lot of time being detailed in the bug report to try to help improve the plane.  I didn't expect to find detailed notes about SPAD.neXt in the Autopilot Feature Guide. Your first link returns a 404 error.  You didn't have to be so rude. I'll figure it out.

**Comment by aguther** [comment]
It seems the docs have been changed, took the link 1:1. I also cannot find the page anymore. So you can use the second link which should contain all you need.
